src/include/duckdb/parser/parsed_data/create_schema_info.hpp

```diff
@@ -25,6 +25,29 @@ struct CreateSchemaInfo : public CreateInfo {
 
 	DUCKDB_API void Serialize(Serializer &serializer) const override;
 	DUCKDB_API static unique_ptr<CreateInfo> Deserialize(Deserializer &deserializer);
+
+	string ToString() const override {
+		string ret = "";
+		switch (on_conflict) {
```

Comment:
This also exists in `SchemaCatalogEntry::ToSQL` but doesn't support the different on conflict clauses - could they be merged as well?